node/setup.go

```diff
@@ -338,6 +338,10 @@ func createBlockchainReactor(
 	metrics *consensus.Metrics,
 ) (*p2p.ReactorShim, service.Service, error) {
 
+	if !cfg.BlockSync.Enable {
+		logger.Error("Blocksync is disabled in the config, but this is not a supported configuration, overriding. This parameter will be completely removed in the next version.")
+	}
```

Comment:
This is fine, but I found something about this wording confusing, so here's an alternative suggestion:

```suggestion
		logger.Error("blocksync.enable = false, but Tendermint no longer allows blocksync to be disabled. This setting is now ignored and will be removed in the next version.")
	}
```